staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -786,6 +786,14 @@ type KubeletConfiguration struct {
 	// +featureGate=KubeletTracing
 	// +optional
 	Tracing *tracingapi.TracingConfiguration `json:"tracing,omitempty"`
+	// localStorageCapacityIsolation enables local storage capacity isolation.
```

Comment:
can you copy the same comment from [pkg/kubelet/apis/config/types.go](https://github.com/kubernetes/kubernetes/pull/111513/files#diff-8a6475c76df7985730ad410e5d1a6caccbbe3ba44853ed773722063cf3625639)

Reply:
sure, I updated the comment